tests/sources/tools/control/single_demo_grasp/test_single_demo_grasp.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import shutil
+import unittest
+import torch
+
+# detectron dependencies
```

Comment:
I think this can be left as it is for now, since the learner provides the basic functionality and there are instructions on how the learner should be used.

Reply:
Hi...those are there since I was not able to directly open and compare outputs of detectron model as torch model to see if the model updates or not. that's why I had to use their functions in tests...I will try to find a better solution for the later versions.